lsp-dart-test-support.el

```diff
@@ -181,9 +181,9 @@ NOTIFICATION is the event notification.")
   "Spawn COMMAND with ARGS on a separated buffer."
   (lsp-dart-test--clean-process-buffer)
   (let ((process-buffer (get-buffer-create lsp-dart-test--process-buffer-name))
-        (project-root (lsp-dart-get-project-root)))
+        (project-rt (lsp-dart-get-project-root)))
     (with-current-buffer process-buffer
-      (setq-local default-directory (or project-root default-directory))
+      (setq-local default-directory (or project-rt default-directory))
```

Comment:
I have temporary rename the variable to `project-rt`, or else the `package-lint` will give me the following error:

```
184:9: error: You should depend on (emacs "28.1") if you need ‘project-root’.
```